core/storage/feedback/gae_models.py

```diff
@@ -72,6 +74,20 @@ class FeedbackThreadModel(base_models.BaseModel):
     has_suggestion = ndb.BooleanProperty(indexed=True, default=False)
     # The number of messages in the thread.
     message_count = ndb.IntegerProperty(indexed=True)
+    # When this thread was last updated.
```

Comment:
Add a comment to explain why this is defined here and that indicates that the superclass field is being overridden.